extension-spring/src/main/java/io/smallrye/openapi/spring/SpringAnnotationScanner.java

```diff
@@ -127,6 +139,15 @@ public OpenAPI scan(final AnnotationScannerContext context, OpenAPI openApi) {
         return openApi;
     }
 
+    private boolean hasRequestMappingMethod(final MethodInfo method, final RequestMethod requestMethod) {
+        if (method.hasAnnotation(SpringConstants.REQUEST_MAPPING)) {
+            AnnotationInstance annotation = method.annotation(SpringConstants.REQUEST_MAPPING);
+            AnnotationValue value = annotation.value("method");
+            return value != null && value.asString().equals(requestMethod.name());
```

Comment:
I think that `value.asEnumArray()` (which returns `String[]`) needs to be used for the `method` property here.

Reply:
Cool ! Thanks :)